arch/tools/cli/core.py

```diff
@@ -1,40 +1,73 @@
 import subprocess
 import os
 import time
-import pkg_resources
-import select
 import sys
 import glob
-from cli.utils import run_docker_compose_ps, print_service_status, check_services_state
+import docker
 from cli.utils import getLogger
 from cli.consts import (
+    ARCHGW_DOCKER_IMAGE,
+    ARCHGW_DOCKER_NAME,
     KATANEMO_LOCAL_MODEL_LIST,
     MODEL_SERVER_LOG_FILE,
     ACCESS_LOG_FILES,
 )
 from huggingface_hub import snapshot_download
+from dotenv import dotenv_values
+
 
 log = getLogger(__name__)
 
+dot_env = dotenv_values(".env")
+
+
+def start_archgw_docker(client, arch_config_file):
```

Comment:
where does the CLI need to be to start arch?

Reply:
here https://github.com/katanemo/archgw/pull/305/files#diff-9fb218171a8bf47ce0a494ac2ccdf5052d2525f6a0223b0ab327e942b3c48c87R132